src/components/MoneyRequestHeader.tsx

```diff
@@ -79,16 +79,11 @@ function MoneyRequestHeader({session, parentReport, report, parentReportAction,
     const isScanning = TransactionUtils.hasReceipt(transaction) && TransactionUtils.isReceiptBeingScanned(transaction);
     const isPending = TransactionUtils.isExpensifyCardTransaction(transaction) && TransactionUtils.isPending(transaction);
 
-    const isRequestModifiable = !isSettled && !isApproved && !ReportActionsUtils.isDeletedAction(parentReportAction);
-    const canModifyRequest = isActionOwner && !isSettled && !isApproved && !ReportActionsUtils.isDeletedAction(parentReportAction);
```

Comment:
I also removed this one, because we aren't really modifying the request in the header. We just need to check for permission to hold/unhold, and delete.

Reply:
Yup it doesn't. We used `isRequestModifiable` for that before, it doesn't have `isActionOwner`.